aip-0001.md

```diff
@@ -7,19 +7,36 @@ Type: Process
 Created: 2022-01-23
 ```
 
-# Summary
+
+# AIP Process
+
+![AIP_process](https://user-images.githubusercontent.com/2584493/153044256-bbab0acf-05d8-42d7-9b13-29cef5dde0ce.png)
```

Comment:
This link should be https://assangedao.org/img/diagram_desktop.svg

Reply:
Doesnt render svg

Reply:
Actually don't worry since the svg won't render.